### PR TITLE
Some changes to the manual

### LINKED ISSUE #1680: TODO in manual: Setup Nightly Builds
I think we should fix the TODO for the manual of our master branch.

http://seqan.readthedocs.org/en/master/Tutorial/HowTo/Recipes/SetupNightlyBuilds.html

@h-2 I'm assigning you since you were working on the Infrastructure part and probably know best whether to integrate it into the Infrastructure part as stated.

### LINKED ISSUE #1782: Maintaining binaries for older OSs
Not an issue, just a question. 

Since the update of SeqAn that now requires GLIBC 2.14 the binaries I have created no longer run on many of the production HPC environments that run Scientific Linux 6 which is GLIB 2.12 which will be unlikely to be updated.

I was just wondering what strategies people are using to provide binary support with previous OSs. e.g. compile apps against old versions of the seqan Library?

### PR DESCRIPTION
**Pull request by h-2**
Fixes #1680 
Partly fixes #1782

### CODE REVIEW

**Comment by h-2** [comment]
@cpockrandt can you check the accuracy of the windows section and followup on this PR with a PR of your own that fills in the gaps? Thanks.

**Comment by h-2** [comment]
@rrahn can you review this today please?

File: manual/source/Infrastructure/Manage/NightlyBuilds.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+The first columns mean that on the 5th minute of the 1st/3rd hour (1:05am/3:05am) of every day, of every month and every week the subsequent command is executed. The variables are described above.
+
+``nice -n 10`` ensures that the cron jobs get a low priority so the system remains responsive. The path after that is the path to your svn checkout. The redirection in the end prevents output from spamming your mail account.
+
+Remember to add all folders to the PATH variable that contain binaries which you have added to ``COMPILERS=``.
+
+Windows
+-------
+
+TODO double check this.
```
Comment (rrahn) [inline manual/source/Infrastructure/Manage/NightlyBuilds.rst]: ?

**Comment by rrahn** [comment]
OK I think, the TODO is meant for @cpockrandt?

**Comment by h-2** [comment]
> OK I think, the TODO is meant for @cpockrandt?

Yep!

**Pull request merged**